src/WebJobs.Script.WebHost/Security/Authorization/AuthUtility.cs

```diff
@@ -42,6 +42,10 @@ public static bool PrincipalHasAuthLevelClaim(ClaimsPrincipal principal, Authori
                 if (level == AuthorizationLevel.Admin)
                 {
                     return true;
+                } else if (isDisabled)
```

Comment:
Spacing issue here.

You can just remove the `else` and have the if statement below the admin check